smpclient/__init__.py

```diff
@@ -86,25 +89,30 @@ async def main():
     ```
     """
 
-    def __init__(self, transport: SMPTransport, address: str):  # noqa: DOC301
+    def __init__(
+        self, transport: SMPTransport, address: str, timeout: float = 60.0
```

Comment:
I realize that we initially had a very long default timeout. But I think that this should be `2.500`s default. Because most devices are local via USB, serial, BLE, or local network, and are expected to respond quickly. Let's discuss.